apps/artist/test/client.view.coffee

```diff
@@ -126,16 +126,16 @@ describe 'ArtistPageView', ->
         @view.followArtists.follow.restore()
         location.href.should.containEql "/log_in?redirect-to="
 
-    describe '#resetArtworks', ->
+  describe '#resetArtworks', ->
 
-      it 'fetches the artworks with the params', ->
-        @view.artworkParams.foo = 'bar'
-        @view.resetArtworks()
-        Backbone.sync.args[0][2].data.foo.should.equal 'bar'
+    it 'fetches the artworks with the params', ->
+      @view.artworkParams.sort = 'title'
+      @view.resetArtworks()
+      Backbone.sync.args[0][2].data.should.containEql 'sort=title'
```

Comment:
These should be `_.last(Backbone.sync.args)[2].data`, (rather than the `[0]`).

The reason for this is initializing the view (in the `beforeEach` block), is already triggering an initial fetch, which is what you're examining with `[0]`. It's good practice to use `_.last` in these cases, when you're initiating a fetch from within your spec, to avoid having to wade thru fetches that happened during the test setup (such as in a `before` or initializer).

I'm not 100% sure why the original spec here used `[0]` and presumably passed, but I'm positive if you switch to `_.last` these will pass as expected :)